bin/rule.ml

```diff
@@ -97,21 +97,23 @@ let pp_direction fmt = function
   | `To_md -> Fmt.pf fmt "--direction=to-md"
   | `To_ml -> Fmt.pf fmt "--direction=to-ml"
 
+let pp_section fmt s = Fmt.pf fmt "-s %S" s
```

Comment:
I think it'd be better to use the long form here: `--section=%S`

Reply:
No. This will result in `--section="abc"` which will be parsed as 2 atoms by dune ("--section=" and "abc").

Reply:
My point was about the output rule being more readable so we can go for `--section` and `"abc"` can't we?

Reply:
Yes that's better.